test/chart/AccessibilityLayer.spec.tsx

```diff
@@ -172,6 +195,83 @@ describe('AccessibilityLayer', () => {
     expect(mockMouseMovements.mock.instances).toHaveLength(0);
   });
 
+  test('Left/right arrow pays attention to if the XAxis is reversed', () => {
+    const mockMouseMovements = vi.fn();
+
+    const { container } = render(
+      <AreaChart width={100} height={50} data={data} accessibilityLayer onMouseMove={mockMouseMovements}>
+        <Area type="monotone" dataKey="uv" stroke="#ff7300" fill="#ff7300" />
+        <Tooltip />
+        <Legend />
+        <XAxis dataKey="name" reversed />
+        <YAxis />
+      </AreaChart>,
+    );
+
+    const svg = container.querySelector('svg');
+    assertNotNull(svg);
+    const tooltip = container.querySelector('.recharts-tooltip-wrapper');
+
+    expect(tooltip?.textContent).toBe('');
+    expect(mockMouseMovements.mock.instances).toHaveLength(0);
```

Comment:
```suggestion
    expect(mockMouseMovements).toHaveBeenCalledTimes(0);
```

This will show better errors if it fails

Reply:
I updated this use of `.toHaveLength`, and all the other cases I had written like it, to use `.toHaveBeenCalledTimes`